common/logisticspipes/modules/ModuleActiveSupplier.java

```diff
@@ -299,30 +305,37 @@ private void createPatternRequest(IInventoryUtil invUtil) {
 		}
 	}
 
-	private void createSupplyRequest(IInventoryUtil invUtil) {
+	private void createSupplyRequest(IInventoryUtil invUtil)
+	{
 		_service.getDebug().log("Supplier: Start calculating supply request");
+		//System.out.println("Supplier: Start calculating supply request");
```

Comment:
Don't leave these in, especially when there's already a log message saying the same thing right above